pkg/controller/model_binding_pair.go

```diff
@@ -288,6 +289,45 @@ func buildModelBindingPair(mbPair *types.ModelBindingPair) *types.ModelBindingPa
 					}
 				}
 			}
+
+			// Does this namespace contain any generic components
+			genericComponents := mbPair.Model.Spec.GenericComponents
+			for _, generic := range genericComponents {
+				_, found := compNames[generic.Name]
+				if found {
+					namespace, _ := util.GetComponentNamespace(generic.Name, mbPair.Binding)
+					genericLabels := util.GetManagedBindingLabels(mbPair.Binding, mc.Name)
```

Comment:
Do we let the user add their own labels to the generic component?

Reply:
Same comment as above.  No, originally that was the plan but Mike Gilbode said that was not needed for the initial deliverable